duckduckgo/iOS
# Broken site reporting reference test

Task/Issue URL: https://app.asana.com/0/0/1203476103622317/f


**Description**:
Add broken site reporting reference tests

**Steps to test this PR**:
1. Run tests, see if they work
2. Report broken site and check if it's still working (adding a breakpoint into the request at the send method on BrokenSiteInfo to check the params should be good enough

**Device Testing**:
* [] iPhone 14 Pro
* [] iPad

**OS Testing**:

* [] iOS 14
* [] iOS 15
* [] iOS 16

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
